privaterelay/settings.py

```diff
@@ -421,6 +421,8 @@ def _get_initial_middleware():
 LANGUAGES = settings.LANGUAGES + [
     ('zh-tw', 'Chinese'),
     ('zh-cn', 'Chinese'),
+    ('fy', 'Frisian'),
+    # ('fy-nl', 'Frisian'),
```

Comment:
Adding Frisian should be in a different PR

Reply:
Good catch. this was a code snippet I was testing for a different bug. I should remove this.

Reply:
Done on 4321891